### PR TITLE
My wishlist

### PR DESCRIPTION
**Pull request by deepak29-git**
1.Added Navigation
2. Added My Wishlist page
https://user-images.githubusercontent.com/88363801/153470620-a1b395f3-aee4-40ab-a08e-7e27d56ae548.mp4

### CODE REVIEW

File: Cart-management/cart.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    width: 25rem;
+    padding:2rem;
+    border-radius: 5px;
+
+}
+
+.space-between{
+    display:flex;
+    justify-content: space-between;    
+}
```
Comment (Vishruta-Patil) [inline Cart-management/cart.css:17]: include this class in the utility css

File: Product-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* center text */
+.center {
+  display: flex;
+  justify-content: center;
+}
```
Comment (Vishruta-Patil) [inline Product-page/product-page.css:41]: include both the classes in the utility css

**Review by Vishruta-Patil — APPROVED** [review]
Everything else looks good.

File: Cart-management/cart.css (hunk 1 above)
Comment (deepak29-git) [reply Cart-management/cart.css:17]: done

File: Product-page/product-page.css (hunk 2 above)
Comment (deepak29-git) [reply Product-page/product-page.css:41]: done

**Review by shivanigangadharan — APPROVED** [review]
Can give some amount of space on the left (the 1st product card is sticking to the boundary)
Rest of it looks good to merge. 👍

**Pull request merged**